WhirlyGlobeSrc/WhirlyGlobeLib/src/ParticleSystemLayer.mm

```diff
@@ -55,7 +55,7 @@ - (void)cleanup
     [self performSelector:@selector(cleanup) withObject:nil afterDelay:kWKParticleSystemCleanupPeriod];
 }
 
-- (void)shutdown
+- (void)teardown
```

Comment:
@mousebird  Similar confusion here. Can we merge "cleanup" and "teardown" methods in a single one?